Automattic/pocket-casts-android
# Display Cover story

## Description

This adds cover story.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3054_2163

> [!note]
> Edge to edge is not supported ATM due to the legacy code. Enabling it would break the UI in the whole app. I'll work on supporting it as a separate task.

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced you should see the cover story as the first story.

## Screenshots or Screencast 

### Preview

| Regular | Tablet |
| - | - |
| ![reg](https://github.com/user-attachments/assets/495e4a10-cfb9-47c5-b4dd-d27b28fbaa5b) | ![tab](https://github.com/user-attachments/assets/38061313-8d1c-41a8-a9e2-c9d1046f8b83) |

### Animation

https://github.com/user-attachments/assets/254f03b0-dc92-4ab6-9740-e3a1555020c9

## Checklist
 
#### I have tested any UI changes...
